components/chart.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Chart"
+description: "A simple component that displays useful information"
+---
+
+```tsx App.tsx
+import { QuillProvider, Chart } from "@quillsql/react";
+
+function App() {
+	return (
+		<QuillProvider
+			organizationId={organizationId}
+			publicKey={API_KEY}
+			authToken={user.jwt}
```

Comment:
auth token doesn't exit

Reply:
Yeah that was old, removed this in the new version.